Tests/LibWeb/Text/input/abortsignal-abort.html

```diff
@@ -0,0 +1,19 @@
+<script src="include.js"></script>
+<script>
+     asyncTest(async done => {
+        function testSignal(signal) {
+            return fetch("./basic.html", { signal })
+                .then(() => {
+                    println("FAIL");
+                })
+                .catch(error => {
+                    println(`Aborted: ${signal.aborted}`);
+                    println(`Reason: "${error}"`);
+                });
+            }
+
+            testSignal(AbortSignal.abort())
```

Comment:
the formatting here looks off. does prettier like this file?

Reply:
Not sure what happened but your latest push seems to have deleted the file instead of reformatting it 😅